service/src/main/kotlin/fi/espoo/evaka/emailclient/EvakaEmailMessageProvider.kt

```diff
@@ -526,7 +527,7 @@ $unsubscribeEn
                     if (event.period.end != event.period.start) {
                         period += "-${event.period.end.format(format)}"
                     }
-                    "<li>$period: ${event.title}</li>"
+                    "<li>$period: ${HtmlEscape.escapeHtml5(event.title)}</li>"
```

Comment:
wonder if escapeHtml5xml would be better here too?

Reply:
It probably doesn't matter since we can assume e-mail clients support HTML entities properly